### PR TITLE
 Fix onChange firing even when on last element with no infinite loop

### PR DESCRIPTION
**Pull request by Tomassito**
onChange callback was firing continuously even when no looping was set

### CODE REVIEW

File: src/__tests__/Carousel.js [hunk 1]
```diff
@@ -1,42 +1,42 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { shallow, mount } from 'enzyme';
```
Comment (sezna) [inline src/__tests__/Carousel.js]: It looks like you also re-formatted a lot of the code with your local editor settings. Could you undo all the whitespace changes and just retain the actual code changes?

Comment (Tomassito) [reply src/__tests__/Carousel.js]: Tried to stick just to logic changes with no formatting adjustments in the latest commit.

**Comment by sezna** [comment]
@Tomassito  It looks like @leandrowd  has added a prettier configuration. You could pull and run prettier to fix this.

**Comment by leandrowd** [comment]
Closing due to lack of activity... feel free to submit again once all issues are resolved.

**Pull request closed without merging**